src/platform/x86_64/vmx/include/vm_vcpu_context.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
```

Comment:
No, it is not taken from any other code base. This is a simple vm context layer created for Composite, I used the `pragma once` to avoid the frequent `#define` style